app/assets/scss/_text.scss

```diff
@@ -0,0 +1,9 @@
+
+.lede {
```

Comment:
It's a literature thing meaning a leading paragraph http://en.wikipedia.org/wiki/Lead_paragraph. Was a new word to me too but apparently it's used elsewhere in GDS.